cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -131,6 +155,64 @@ spec:
             path: "/etc/ssl/certs/ca-certificates.crt"
 ```
 
+### Auto-Discovery Setup
+
+As of today, docker images including the support for `--node-group-auto-discovery` is not yet published to official repository.
+Please checkout the latest source of this project locally and run `REGISTRY=<your docker repo> make release` to build and push an image yourself.
+Then, a manifest like below would run a cluster-autoscaler which auto-discovers ASGs tagged with `k8s.io/cluster-autoscaler/enabled` to be node groups.
+Please notice that there're no `--nodes` flags passed to cluster-autoscaler anymore.
```

Comment:
Thanks 👍 

> Is there a reason we're not support both the flag and the manifest for at least 1 major version?

I'm afraid if I'm following you correctly but the `--nodes` flag is not removed. It's just that you can specify either `--nodes` or `--node-group-auto-discovery`, but not both.